### PR TITLE
Updated README for the repo consistency project

### PR DESCRIPTION
**Pull request by paperclypse**
For context:
https://docs.google.com/document/d/1JIog8rJL9MT6_AbIji8VK_RT3mOuIxC2Xb_xQsVksKk/edit#

https://newrelic.atlassian.net/browse/DOC-6121

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +57,11 @@ If you're compiling your JavaScript and can't control the final `require` order,
     $ node -r newrelic your-program.js
     $ node --require newrelic your-program.js
 
+## Getting Started
+
 For more information on getting started, [check the Node.js docs](https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs).
 
-## Using the API
```
Comment (michaelgoin) [inline README.md:62]: This section is specific to using the agent API. As such, perhaps it should be nested under "Usage" if it is important to have that as a header but I would not remove reference to the API for the content below it.

**Review by michaelgoin — CHANGES_REQUESTED** [review]
Mostly looks good.

One comment on the "using the api" section being specific to usage of the API and not using the agent in general. As such, we want the specific header (or something similar) whether or not you want to add an additional "Usage" section.

File: README.md (hunk 1 above)
Comment (paperclypse) [reply README.md:62]: I added a "Using the API" section under the Usage header. I think your point is super valid, but I do think it's important to keep the Usage header for consistency with other repos.

**Review by michaelgoin — APPROVED** [review]
👍

**Pull request merged**